front-end/src/lib/apiUtils.ts

```diff
@@ -1,5 +1,8 @@
 import { AnalyzerType, FileInformation, FileTimespan, MinMax } from '@types';
 
+export const isElectron = typeof process !== 'undefined' && process.versions?.electron;
```

Comment:
Definitely not the spot for this. This should only be logic to fetch from the Api

Reply:
moved into a new electronUtils.ts file